### DISCUSSION TITLE
I still need help...

### DISCUSSION THREAD

**Discussion by birdoneadmin** [community/community#64850] [Repositories]
### Select Topic Area

Question

### Body

I understand this is just a link, however it is serverly affecting the workflow of our company. Please help. #64480

**Comment by queenofcorgis** [comment]
Hi @birdoneadmin ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was closed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
